neqo-http3/src/connection.rs

```diff
@@ -53,6 +53,22 @@ where
     pub priority: Priority,
 }
 
+pub enum WebTransportSessionAcceptAction {
+    Accept,
+    Reject,
+    Redirect
+}
+
+impl ::std::fmt::Display for WebTransportSessionAcceptAction {
+    fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
+        match self {
+            WebTransportSessionAcceptAction::Accept => write!(f, "Accept"),
+            WebTransportSessionAcceptAction::Reject => write!(f, "Reject"),
+            WebTransportSessionAcceptAction::Redirect => write!(f, "Redirect"),
```

Comment:
```suggestion
            WebTransportSessionAcceptAction::Accept => f.write_str("Accept"),
            WebTransportSessionAcceptAction::Reject => f.write_str("Reject"),
            WebTransportSessionAcceptAction::Redirect => f.write_str("Redirect"),
```

Or even
```rust
        f.write_str(match self {
            WebTransportSessionAcceptAction::Accept => "Accept",
            WebTransportSessionAcceptAction::Reject => "Reject",
            WebTransportSessionAcceptAction::Redirect => "Redirect",
        })
```

No need to use macros for this one.